ArduCopter/avoidance.cpp

```diff
@@ -0,0 +1,19 @@
+#include "Copter.h"
+
+void Copter::low_alt_avoidance()
```

Comment:
let's put a comment above this method, "// check if avoidance should be enabled based on alt" or something similar..